src/components/RegionVaccinationBlock/RegionVaccinationBlock.tsx

```diff
@@ -1,20 +1,3 @@
-import React from 'react';
 import VaccinationBlock from './VaccinationBlock';
-import StateVaccinationBlock from './StateVaccinationBlock';
-import { fail } from 'common/utils';
-import { Region, County, State, MetroArea } from 'common/regions';
 
-const RegionVaccinationBox: React.FC<{ region: Region }> = ({ region }) => {
-  if (region instanceof County || region instanceof MetroArea) {
-    return <VaccinationBlock region={region} />;
-  } else if (region instanceof State) {
-    return <StateVaccinationBlock region={region} />;
-  } else {
-    fail(
-      `RegionVaccinationBox: missing implementation for region with fips ${region.fipsCode}`,
-    );
-    return null;
-  }
-};
-
-export default RegionVaccinationBox;
+export default VaccinationBlock;
```

Comment:
We don't need this anymore right?

Reply:
We don't, but I want to keep it until we launch just in case the decision is reversed - I will clean up early next week I promise!